examples/svelte/svelte.config.js

```diff
@@ -0,0 +1,39 @@
+import adapter from '@sveltejs/adapter-static';
+import preprocess from 'svelte-preprocess';
+import path, { resolve } from 'path';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	// Consult https://github.com/sveltejs/svelte-preprocess
+	// for more information about preprocessors
+	preprocess: preprocess(),
+
+	kit: {
+		adapter: adapter(),
+
+		// hydrate the <div id="svelte"> element in src/app.html
+		target: '#svelte',
+		vite: {
+			ssr: {
+				noExternal:
+					process.env.NODE_ENV !== 'development' ? ['lodash', 'xstate', 'style-dictionary'] : []
```

Comment:
For my understanding, why were these three packages needed for `dev`?

Reply:
From reading the [documentation](https://vitejs.dev/guide/ssr.html#ssr-externals) the ssr in vite is still experimental. It tries to externalize dependencies depending if they have a valid ESM entry point. However, when building for production, lodash, xstate and style-dictionary would not externalize correctly leading to errors like seen below. I added this line in to prevent this error and have it build properly.


```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/ehhanche/programming/work/spark/amplify-ui/node_modules/lodash/includes' imported from /Users/ehhanche/programming/work/spark/amplify-ui/examples/svelte/.svelte-kit/output/server/entries/pages/ui/components/authenticator/sign-in-with-username/index.svelte.js
Did you mean to import lodash/includes.js?
```

Reply:
Got it, thanks 👍